lisa/notifiers/env_stats.py

```diff
@@ -82,7 +82,7 @@ def _process_test_result_message(self, test_result: TestResultMessage) -> None:
         result_info = self._test_results.get(test_result.id_, None)
         if not result_info:
             result_info = TestResultInformation(
-                id=test_result.id_, name=test_result.name
+                id=test_result.id_, name=test_result.full_name
```

Comment:
FYI: `name` was renamed to `full_name` so that `name` could be used for relative name instead of the full name.